easybuild/easyconfigs/i/imkl/imkl-2018.1.163-iccifort-2018.1.163-GCC-6.4.0-2.28-serial.eb

```diff
@@ -0,0 +1,38 @@
+# This is an easyconfig file for EasyBuild, see https://easybuilders.github.io/easybuild/
+
+name = 'imkl'
+version = '2018.1.163'
+versionsuffix = '-serial'
+
+homepage = 'https://software.intel.com/en-us/intel-mkl/'
+description = """Intel Math Kernel Library is a library of highly optimized,
+ extensively threaded math routines for science, engineering, and financial
+ applications that require maximum performance. Core math functions include
+ BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more."""
+
+toolchain = {'name': 'iccifort', 'version': '2018.1.163-GCC-6.4.0-2.28'}
+
+source_urls = ['https://registrationcenter-download.intel.com/akdlm/irc_nas/tec/12384/']
+sources = ['l_mkl_%(version)s.tgz']
+checksums = ['f6dc263fc6f3c350979740a13de1b1e8745d9ba0d0f067ece503483b9189c2ca']
+
+dontcreateinstalldir = 'True'
```

Comment:
This should be:

```python
dontcreateinstalldir = True
```

Also this is missing (in comparison with existing `imkl` 2018.1.163 easyconfigs):

```python
components = ['intel-mkl']
```

Reply:
@bartoldeman Are you up for making these trivial changes, so we can get this merged?

Reply:
ok